docs/content/api-reference/_index.md

```diff
@@ -6,17 +6,17 @@ chapter = false
 pre = "<b>5. </b>"
 +++
 
-Ogmios as a JSON-WSP service is entirely described using [JSON Schema - Draft 7](https://json-schema.org/). This can be fed into various tools to generate code, data-types or definitions in many languages. In particular, the server schema can be visualized [here](http://localhost:1313/api/interfaces/_cardano_ogmios_schema.Ogmios.html). 
+Ogmios as a JSON-WSP service is entirely described using [JSON Schema - Draft 7](https://json-schema.org/). This can be fed into various tools to generate code, data-types or definitions in many languages. In particular, the server schema can be visualized [here](/api/interfaces/_cardano_ogmios_schema.Ogmios.html).
```

Comment:
:facepalm: ... Guess I was too tired to think about removing the localhost! 

What do you mean "it works on my local machine!" :smile: